moveit_ros/planning_interface/moveit_cpp/src/demo.cpp

```diff
@@ -0,0 +1,121 @@
+/*********************************************************************
```

Comment:
turn this into a unit test. demos only belong in the tutorials

Reply:
Removed this for now, I'll make a new one